src/pages/support/tracing.md

```diff
@@ -1,9 +1,10 @@
 # Tracing Adobe I/O Events
 
-For help in debugging your webhook-based event-registrations, Adobe offers Debug Tracing. Using Debug Tracing, you can inspect the activity between the Adobe service and your integration to see what data is being sent and how your integration is responding.
-
+To troubleshoot issues with webhook-based event-registrations, Adobe offers Debug Tracing feature. Using Debug Tracing, you can inspect the activity between the Adobe service and your integration to see what data is being sent and how your integration is responding.
 Each trace comprises of the event delivery HTTP request (sent by I/O Events), and response by the consumer application. This information can be of critical importance to your debugging efforts.
 
+> Note that the maximum size of 'request-body' and 'response-body' in a trace is 2 KB. If any of these fields are larger than 2 KB, then they are trimmed at 2 KB boundary. The event payload that was delivered to your webhook is not trimmed.
```

Comment:
Do you think  a screenshot of a trimmed request/response will here?

Reply:
Also this is how it is getting rendered. Should we simply use the `<InlineAlert variant="info" slots="text"/>` version of Gatsby instead?

<img width="797" alt="debug-tracing" src="https://user-images.githubusercontent.com/6910192/152527122-a6aeef60-78ec-4f5c-9360-94b8daccbd0a.png">

Reply:
Inline-block looks good. Thanks! 

![image](https://user-images.githubusercontent.com/5928555/152535631-94ba2483-a8b7-4ce5-b48a-7aae6d9c88f4.png)

Reply:
>Do you think a screenshot of a trimmed request/response will here?

I think adding a screenshot for >2KB case adds unnecessary complexity. It is not adding any value. Here's a screenshot for reference - 

![image](https://user-images.githubusercontent.com/5928555/152536108-37c95e1d-c36d-4fc1-8057-d58f81c84fbf.png)

Reply:
Right, it is not helping!